modules/@apostrophecms/schema/ui/apos/mixins/AposInputMixin.js

```diff
@@ -19,6 +19,10 @@ export default {
         return [];
       },
       type: Array
+    },
+    docId: {
```

Comment:
if this is only used in the slug input it doesn't need to be in the mixin.

Reply:
Every component gets it in all cases, so I thought it has to be defined in the mixin. The same for `followsValue` hereunder. What do you think? Should I move these into `AposInputSlug` only even if, potentially, later another component will use them?